ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -1570,6 +1570,28 @@ check_parent_attrs_against_processing(enum pkcs11_mechanism_id proc_id,
 			return PKCS11_CKR_KEY_FUNCTION_NOT_PERMITTED;
 		}
 		break;
+	case PKCS11_CKM_RSA_PKCS:
+	case PKCS11_CKM_MD5_RSA_PKCS:
+	case PKCS11_CKM_SHA1_RSA_PKCS:
+	case PKCS11_CKM_SHA256_RSA_PKCS:
+	case PKCS11_CKM_SHA384_RSA_PKCS:
+	case PKCS11_CKM_SHA512_RSA_PKCS:
+	case PKCS11_CKM_SHA224_RSA_PKCS:
```

Comment:
prefer move this line between sha1 and sha256 cases, for consistency.

Reply:
OK